counter/README.md

```diff
@@ -0,0 +1,8 @@
+# Counter
```

Comment:
I really wanted to call it a "counterexample" :-)
Plus, I think a small explanation of the intent would help.

```suggestion
# A counterexample
A set of progressively more complex examples, demonstrating ways to compose complex UIs out of simpler ones, using an inc/dec counter as base.
```